grafana-plugin/src/pages/incidents/Incidents.tsx

```diff
@@ -66,9 +68,10 @@ interface IncidentsPageState {
   showAddAlertGroupForm: boolean;
   isSelectorColumnMenuOpen: boolean;
   isHorizontalScrolling: boolean;
+  isFirstIncidentsFetchDone: boolean;
 }
 
-const POLLING_NUM_SECONDS = 15;
+const POLLING_NUM_SECONDS = 5;
```

Comment:
any reason why this is set to 5 instead of 15? might run into issues with 5 if you're performing any selections on the alert groups and once new data comes in might revert any current state.

Reply:
agree, 5 secs looks to small

Reply:
This is only "try every 5 secs". If previous call is pending, it's cancelled immediately and it does nothing. In general I prevented race conditions that led to incorrect alert groups state. For small stacks, when there are not many alert groups, why user should wait 15 secs to see the change applied by another user?

Reply:
brought back 15secs